submit_for_testing.py

```diff
@@ -398,7 +398,7 @@ def main():
             logger.debug(testpath)
             logger.debug(test)
             container = client.containers.run(
-                image="lavasoftware/amd64-lava-server:2019.12",
+                image="lavasoftware/amd64-lava-server:2020.07",
```

Comment:
maybe we should use 'latest' here?

Reply:
isn't it better to update this when we see/find issues with the current one we have?

Reply:
It's only used for checking the generated jobs, so latest should be the target. If you want to test against actual instance, it should be done differently

Reply:
OK I'll update that